_includes/footer.html

```diff
@@ -32,7 +35,7 @@ <h2 class="text-xl text-yellow-600">Community</h2>
                 </a>
             </div>
             <p class="text-center w-full mt-10">
-                &copy; 2019 <strong>Kadalu Authors</strong>. All rights reserved.
+                &copy; 2020 <strong>Kadalu Software Pvt Ltd.</strong>. All rights reserved.
```

Comment:
2019-2020?

Reply:
I have noticed that on the website, people keep only latest year. At least on many of the company sites.

Reply:
ack